service/hook/endpoint.go

```diff
@@ -199,6 +199,10 @@ func (c *Client) HTTPHandler(w http.ResponseWriter, r *http.Request) {
 
 		if len(webhookMetricsList) > 0 {
 			for _, webhookMetrics := range webhookMetricsList {
+				if webhookMetrics == nil {
```

Comment:
This change seems to be valid, I wonder how it is related to this PR or how you come across this?